src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -316,3 +316,30 @@ def load_command_table(self, _):
             'rollback',
             'aks_mesh_upgrade_rollback',
             supports_no_wait=True)
+
+    # AKS approuting commands
+    with self.command_group('aks approuting', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
+        g.custom_command(
+            'enable',
+            'aks_approuting_enable')
+        g.custom_command(
+            'disable',
+            'aks_approuting_disable', confirmation=True)
+        g.custom_command(
+            'update',
+            'aks_approuting_update')
+
+    # AKS approuting dns-zone commands
+    with self.command_group('aks approuting zone', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
+        g.custom_command(
+            'add',
+            'aks_approuting_zone_add')
```

Comment:
We usually recommend naming the creation commands as `create` instead of `add`. What is the reason for designing it as `add`?

Reply:
We are using `add` since we are adding already created DNS Zone resources to the app routing addon.